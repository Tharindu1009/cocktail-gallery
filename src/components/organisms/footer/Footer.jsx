import './styles/footer.scss';
import { Typography } from '@mui/material';

function Footer() {
    return (
        <div data-testid="footer" className='footer'>
            <Typography variant="caption" display="block" className='copyright-text '>
                Copyright Â©2023
            </Typography>
        </div>
    )
}

export default Footer;