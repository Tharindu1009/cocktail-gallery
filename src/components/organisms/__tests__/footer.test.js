import { render, cleanup } from "@testing-library/react";
import TestRenderer from 'react-test-renderer';
import Footer from "../footer/Footer";

describe('Footer', () => {
    afterEach(() => {
        cleanup();
    })

    it('should render footer component correctly', () => {
        const { getByTestId } = render(<Footer />);
        const footerElement = getByTestId('footer');
        expect(footerElement).toBeInTheDocument();
        expect(footerElement).toHaveTextContent('Copyright Â©2023');
    });

    it('matches footer snapshot', () => {
        const tree = TestRenderer.create(<Footer />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});