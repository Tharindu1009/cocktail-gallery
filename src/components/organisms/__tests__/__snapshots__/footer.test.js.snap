// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches footer snapshot 1`] = `
<div
  className="footer"
  data-testid="footer"
>
  <span
    className="MuiTypography-root MuiTypography-caption copyright-text  css-11idbvn-MuiTypography-root"
  >
    Copyright Â©2023
  </span>
</div>
`;
